graylog2-server/src/main/java/org/graylog/plugins/views/storage/migration/state/actions/MigrationActionsImpl.java

```diff
@@ -196,12 +196,13 @@ public boolean dataNodeStartupFinished() {
 
     @Override
     public void startRemoteReindex() {
+        final String allowlist = getStateMachineContext().getActionArgumentOpt("allowlist", String.class).orElseThrow(() -> new NullPointerException("allowlist has tp be provided"));
```

Comment:
The form will return an empty string if no setting is provided, so the validation does not really work. Also, there's a typo (has t`o`)

Reply:
but I tagged the field as "required" in the form?

Reply:
hmm.. i could submit it with an empty value